File: DomZaStaraLicaApi/Endpoints/Napomena/GetAll/NapomenaGetAllResponse.cs
namespace DomZaStaraLicaApi.Endpoints.Napomena.GetAll
{
    public class NapomenaGetAllResponse
    {
        public List<NapomenaGetAllResponseNapomena> Napomene { get; set; }  
    }
    public class NapomenaGetAllResponseNapomena
    {
        public int NapomenaId { get; set; }
        public string Opis { get; set; }
        public bool Prioritet { get; set; }
        public DateTime DatumPostavke { get; set; }
        public int ZaposlenikId { get; set; }
        public int KorisnikDomaID { get; set; }
        public int VrstaNapomeneId { get; set; }
    }
}

File: frontend/src/app/pregled-napomena/napomenaGetAllResponse.ts
export interface NapomenaGetAllResponse{
napomene:NapomenaGetAllResponseNapomena[]
}

export interface NapomenaGetAllResponseNapomena{
  napomenaId: number
  opis: string
  prioritet: boolean
  datumPostavke: Date
  isAktivna:boolean
  zaposlenikId: number
  korisnikDomaID: number
  vrstaNapomeneId: number
}

